crates/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -119,7 +157,22 @@ impl Environment for ManagedEnvironment {
         flox: &Flox,
         contents: String,
     ) -> Result<EditResult, EnvironmentError2> {
-        todo!()
+        let mut generations = self.generations().writable(flox.temp_dir.clone()).unwrap();
+        let mut temporary = generations.get_current_generation().unwrap();
+
+        let result = temporary.edit(flox, contents)?;
+        let metadata = "description".to_string();
```

Comment:
nonblocking: do you mean to fill in a better description?

Reply:
yes, somethink more precise than "description"